maps/sccv_horizon/code/sccv_horizon_overmap.dm

```diff
@@ -164,3 +165,41 @@
 	icon_keyboard = null
 	circuit = null
 
+// --- raskolnikov
+
+/obj/effect/overmap/visitable/ship/landable/raskolnikov
+	name = "Raskolnikov"
+	class = "ICV"
+	designation = "Raskolnikov"
+	desc = "\
+		A standard-sized transport shuttle manufactured by Hephaestus, the Finland-class is commonly used all around the Spur. \
+		It is cheap, easy to maintain, and spare parts can mostly be produced without any specialized machining tools. \
+		"
+	shuttle = "Raskolnikov"
```

Comment:
I might be missing it, but is this going to be the only name this shuttle uses? I can change this in my remap PR, but I do not think the shuttle should use the same name because it would give away the gamemode nearly instantly when it is seen on the overmap.